### PR TITLE
use currency shortcode instead

### PR DESCRIPTION
**Pull request by reza-fs**

### CODE REVIEW

File: src/javascript/_common/base/currency_base.js [hunk 1]
```diff
@@ -62,12 +62,19 @@ const CryptoConfig = (() => {
...
+        EURS: { display_code: 'EURS',  name: localize('STASIS Euro'),      min_withdrawal: 0.02, pa_max_withdrawal: 2000,    pa_min_withdrawal: 10 },
+        PAX:  { display_code: 'PAX',   name: localize('Paxos Standard'),      min_withdrawal: 0.02, pa_max_withdrawal: 2000,    pa_min_withdrawal: 10 },
+        TUSD: { display_code: 'TUSD',  name: localize('True USD'),      min_withdrawal: 0.02, pa_max_withdrawal: 2000,    pa_min_withdrawal: 10 },
+        USDC: { display_code: 'USDC',  name: localize('USD Coin'),      min_withdrawal: 0.02, pa_max_withdrawal: 2000,    pa_min_withdrawal: 10 },
+        UST:  { display_code: 'USDT',  name: localize('Tether Omni'),        min_withdrawal: 0.02,  pa_max_withdrawal: 2000, pa_min_withdrawal: 10 },
+        eUSDT:  { display_code: 'eUSDT', name: localize('Tether ERC20'),        min_withdrawal: 0.02,  pa_max_withdrawal: 2000, pa_min_withdrawal: 10 },
+        USB:  { display_code: 'USB',   name: localize('Binary Coin'),   min_withdrawal: 0.02,  pa_max_withdrawal: 2000, pa_min_withdrawal: 10 },
```
Comment (reza-fs) [inline src/javascript/_common/base/currency_base.js]: @negar-binary Would you please help confirming `min_withdrawal`, `pa_max_withdrawal` and `ma_min_withdrawal` for newly added values?

Comment (reza-fs) [reply src/javascript/_common/base/currency_base.js]: I should update the tests with the new values as well.

File: src/javascript/app/pages/user/view_popup/view_popup.js [hunk 2]
```diff
@@ -828,7 +828,7 @@ const ViewPopup = (() => {
 
     const responseProposal = (response) => {
         if (response.error) {
-            if (response.error.code !== 'AlreadySubscribed' && +response.echo_req.contract_id === contract_id) {
+            if (response.error.code !== 'AlreadyScurrency_wrapperubscribed' && +response.echo_req.contract_id === contract_id) {
```
Comment (negarn) [inline src/javascript/app/pages/user/view_popup/view_popup.js:831]: looks like a typo

File: src/javascript/_common/base/currency_base.js [hunk 3]
```diff
@@ -117,7 +127,9 @@ const getPaWithdrawalLimit = (currency, limit) => {
     return limit === 'max' ? 2000 : 10; // limits for fiat currency
 };
 
-const getCurrencyDisplayCode = currency => getPropertyValue(CryptoConfig.get(), [currency, 'display_code']) || currency;
+const unifyUST = currency => /UST/.test(currency) ? 'USDT' : currency; // TODO Remove once BE sends a unique currency
```
Comment (negarn) [inline src/javascript/_common/base/currency_base.js:130]:
```suggestion
const unifyUST = currency => /UST/.test(currency) ? 'USDT' : currency; // TODO Remove once API sends a unique currency
```

**Review by negarn — APPROVED** [review]

**Pull request merged**